config/routes.rb

```diff
@@ -0,0 +1,9 @@
+# Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+Rails.application.routes.draw do
+  resources :users, only: [:index, :show] do
+    resources :posts, only: :show, only: [:index, :show]
```

Comment:
## [OPTIONAL]

- [ ] Nice job on putting routes however I can see a repeated permission for the show route for a post `resources :posts, only: :show, only: [:index, :show]` kindly correct it with `resources :posts, only: [:index, :show]`